features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/utils/custom-validation/custom-validation.js

```diff
@@ -43,6 +43,15 @@ $(document).ready(function() {
         return value.indexOf("{}") == -1
     }, 'Empty curly brackets "{}" are not allowed in context field.');
 
+    $.validator.addMethod('validateUrl', function(value, element){
+        var validUrlRegex = /^(http|https):\/\/(.)+/g;
```

Comment:
Is the regex validate urls starting from http and https with following two forward slashes. with (.)+ are we checking it's should contain one or more . ?